src/components/ComboBox/sgds-combo-box.ts

```diff
@@ -0,0 +1,112 @@
+import { ScopedElementsMixin } from "@open-wc/scoped-elements";
+import { html } from "lit";
+import { property } from "lit/decorators.js";
+import { ref } from "lit/directives/ref.js";
+import { DropdownElement } from "../../base/dropdown-element";
+import { defaultValue } from "../../utils/defaultvalue";
+import { SgdsDropdownItem } from "../Dropdown";
+import { SgdsInput } from "../Input";
+import styles from "./sgds-combo-box.scss";
+
+/**
+ * @summary ComboBox component is used for users to make one or more selections from a list.
+ *
+ * @slot icon - slot for form control icon to be displayed on the right of the input box.
+ *
+ * @event sgds-select - Emitted when the combo box's selected value changes.
```

Comment:
i think its sgds-change instead of sgds-select

Reply:
Oh, it's `sgds-select` cos I removed the `sgds-change` event emitted from `ComboBox` from before. `sgds-select` already emitted from parent element `DropdownElement`

Reply:
should there be an event emitted for initialvalue as well?

Reply:
I think no need? Cos user can set the initial value; whenever initial value changes (user types something else into input box), `sgds-select` will be emitted alr

Reply:
icic ok